examples/nanocoap_server/coap_handler.c

```diff
@@ -97,11 +97,16 @@ static ssize_t _riot_value_handler(coap_pkt_t *pkt, uint8_t *buf, size_t len, vo
     case COAP_PUT:
     case COAP_POST:
     {
```

Comment:
case: - block is not need after change
    was there to create a frame that is now just need if if is true 
```suggestion    
```

Reply:
true that. as that's cosmetic, I propose we fix that in master but let this backport get in as is.

Reply:
#16399